pkg/attribute/mutableBag.go

```diff
@@ -64,6 +64,14 @@ func GetMutableBag(parent Bag) *MutableBag {
 	return mb
 }
 
+// GetFakeMutableBagForTesting returns a Mutable bag based on the specified map
+// Use this function only for testing purposes.
+func GetFakeMutableBagForTesting(v map[string]interface{}) *MutableBag {
+	m := GetMutableBag(nil)
```

Comment:
We don't want to compile in testing.T artifacts in production code. I think it is plenty clear from the method name, what it the intended use.